sklearn/utils/tests/test_sparsefuncs.py

```diff
@@ -94,7 +94,9 @@ def test_incr_mean_variance_axis():
         rng = np.random.RandomState(0)
         n_features = 50
         n_samples = 10
-        data_chunks = [rng.random_integers(0, 1, size=n_features)
```

Comment:
Hello @MechCoder , would you explain why original test use `random_integers` instead of `rand ` here?
I can't figure out :cry: